src/main/java/create_task_use_case/CreateTaskInteractor.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Create Task Interactor Class.
+ * Author: Fardin Faruk
+ * Modified By: Fardin Faruk
+ * Created: Nov 19, 2022
+ * Last Modified: Nov 26, 2022
+ */
+package create_task_use_case;
+
+import entities.Task;
+import entities.TaskFactory;
+
+public class CreateTaskInteractor implements CreateTaskInputBoundary {
+    final CreateTaskDsGateway createTaskDsGateway;
+    final CreateTaskOutputBoundary createTaskOutputBoundary;
+    final TaskFactory taskFactory;
+
+    public CreateTaskInteractor(TaskFactory taskFactory, CreateTaskOutputBoundary createTaskOutputBoundary,
+                                CreateTaskDsGateway createTaskDsGateway){
+        this.taskFactory = taskFactory;
+        this.createTaskOutputBoundary = createTaskOutputBoundary;
+        this.createTaskDsGateway = createTaskDsGateway;
```

Comment:
the "createTask" variable names could be abridged to "outputBoundary" and "dsGateway" since their correlation to create task is clear from the class they were initialized as.

Reply:
Done.